fetch.bs

```diff
@@ -4569,7 +4569,8 @@ steps:
   "<code>blank</code>", then return a new <a for=/>response</a> whose
   <a for=response>status message</a> is `<code>OK</code>`, <a for=response>header list</a> is «
   (`<code>Content-Type</code>`, `<code>text/html;charset=utf-8</code>`) », and
-  <a for=response>body</a> is the empty byte sequence.
+  <a for=response>body</a> is the <a for="body with type">body</a> of the result of
+  <a for=BodyInit>safely extracting</a> the empty byte sequence.
```

Comment:
A shortcut for this would be appreciated, for use in HTML as well. Basically body-from-byte-sequence. It looks like you could use it below too.

Reply:
#1507